code/modules/organs/prosthetics/prosthetics_manufacturer.dm

```diff
@@ -10,6 +10,7 @@
 	eye_flash_mod = 1
 	eye_darksight_range = 2
 	associated_gender = PLURAL
+	retrievable_via_pronoun = FALSE
```

Comment:
is there a reason not to just set `associated_gender = null` instead? i guess this would make it so that selecting the FBP bodytype sets your pronouns to they/them but i don't know if that's actually desirable

Reply:
i'm not sure about selecting the FBP bodytype in character setup changing your character's pronouns to plural though... and if we gated that behind `retrievable_via_pronoun` as well it'd mean `associated_gender` is entirely unused for prostheses, i.e. it'd be identical to just setting `associated_gender = null`.